packages/react/src/pagination/Pagination.tsx

```diff
@@ -76,14 +67,13 @@ export const Pagination = forwardRef<HTMLElement, PaginationProps>(
                       alignItems="stretch"
                       display="flex"
                       justifyContent="center"
-                      py="10"
+                      px="6"
+                      py="8"
                       size="md"
                     >
-                      <Box alignItems="end" display="flex">
-                        <Box asChild>
-                          <IconEllipsis />
-                        </Box>
-                      </Box>
+                      <Icon asChild>
+                        <IconEllipsis />
```

Comment:
Use `icon` prop instead of children for icon only buttons?

Reply:
Vai, did you mean using `Button` component instead of `Icon` for the ellipsis?

Reply:
ow my bad - i thought this was a button - nvm